privacy_policy.md

```diff
@@ -0,0 +1,45 @@
+
+USF Maps Project Privacy Policy
+
+We value your privacy, so we want to be explicit about what kind of personal information is involved when you interact with USF Maps.
+
+This page is for the USF Maps project as a whole. The project as such doesn’t collect any personally identifying information from users.  However, instances of USF Maps that serve a particular region may save log and other information, some of which is personally identifiable. Each instance should have its own privacy policy, linked from its home page.
```

Comment:
We can delete this paragraph for now - once we figure out multiple campuses/white-labeling we can revisit multiple instances language.